README.md

```diff
@@ -167,8 +169,6 @@ ParameterType(
 )
 ```
 
-*Note: Currently the parameter name cannot be customized, so the custom parameters can only be used with the type name, e.g. `{Color}`.*
```

Comment:
This is presumably only for Python. But I'd leave it out of scope in this PR.

Reply:
Precedes Python implementation, was associated with .NET example. Moved to #277